spec.html

```diff
@@ -47616,21 +47616,22 @@ <h1>%WrapForValidIteratorPrototype%.return ( )</h1>
 
         <emu-clause id="sec-iterator.prototype">
           <h1>Iterator.prototype</h1>
-          <p>The initial value of Iterator.prototype is %Iterator.prototype%.</p>
+          <p>The initial value of Iterator.prototype is the Iterator prototype object.</p>
           <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
         </emu-clause>
       </emu-clause>
     </emu-clause>
 
     <emu-clause oldids="sec-%iteratorprototype%-object" id="sec-%iterator.prototype%-object">
-      <h1>The %Iterator.prototype% Object</h1>
-      <p>The <dfn>%Iterator.prototype%</dfn> object:</p>
+      <h1>The Iterator Prototype Object</h1>
```

Comment:
(optional)
Other sections that starts with the "The * prototype object:" list has a title with "Properties of the Foo Prototype Object" style.
So, using "Properties of the Iterator Prototype Object" as the title here might fit more.

Of course the current "The Iterator Prototype Object" title also makes sense.

Reply:
I prefer this way, but I'll change it for consistency for now.